OxfordBodleian/BodNewMSS/BDLaethg47.xml

```diff
@@ -159,8 +159,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <width>87–90</width>
                                     </dimensions>
                                     <measure unit="g" type="weight">50</measure> <!--without case-->
-                                    <measure unit="g" type="weight">69</measure> <!--with case-->
-                                   <!-- which one?-->
+                             <!--       <measure unit="g" type="weight">69</measure> <!-\-with case-\->-->
```

Comment:
I don't remember what was decided according to the weight. Shall we use the weight with or without case?

Reply:
We agreed to use the weight without case